pkg/controller/vrouter/vrouter_controller.go

```diff
@@ -469,6 +469,8 @@ func (r *ReconcileVrouter) Reconcile(request reconcile.Request) (reconcile.Resul
 			(&daemonSet.Spec.Template.Spec.InitContainers[idx]).Command = instanceContainer.Command
 		}
 		if container.Name == "vrouterkernelinit" {
+			reqLogger.Info("I detected vrouterkernelinit")
```

Comment:
I would either delete these log entries, or make them more descriptive, e.g.: "Adding vrouterkernelinit to init containers in daemonset pod spec"

Reply:
And if you plan to leave it, then maybe change from Info to Debug

Reply:
Done, sorry